src/openrct2-ui/windows/Sign.cpp

```diff
@@ -156,6 +156,7 @@ class SignWindow final : public Window
                 if (tileElement == nullptr)
                 {
                     Close();
+                    return;
```

Comment:
Nice catch. This would've always caused crashes before this change, since `tileElement` gets used below.